router.yml

```diff
@@ -11,7 +11,12 @@
   roles:
     - role: debian_upgrade_first
 
-- import_playbook: postgresql.yml
+- hosts:
+    - database
+    - router-api
```

Comment:
Why is it needed on both `database` and `router-api` host? `router` should use either `database` or `router-api` as a database host, but not both.